tests/acceptance/shared/util/constants.go

```diff
@@ -0,0 +1,71 @@
+package util
```

Comment:
Constants files and types, and utils, etc are anti patterns in Go. These values should be defined as close to where they're used as possible.

Reply:
Since in our use case here is to be a framework and we have unfornately some global variables , so to not polute files and repeat myself creating unnecessary variables acrooss it, we decided to use a file to gather that and not duplicate.

What i did now to improve that based on you comment was : 

- having on the utils package ( i dont like the name , but came no other better name in my mind to this ) only the global vars that are used across packages 


- create another constant file for having the constants related to that package in the same place.

Reply:
cleaned and changed, tried to remove most possible.